internal/handlers/flags_test.go

```diff
@@ -0,0 +1,117 @@
+package handlers_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/placer14/gof-server/internal/handlers"
+	"github.com/placer14/gof-server/internal/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+type respValueType struct {
+	Value any `json:"value"`
+}
+
+func TestGetStringValueHandler(t *testing.T) {
+	m := storage.NewInMemoryStorage()
+	m.SetString("hello", "world")
+	ctx := context.Background()
+	ctx = context.WithValue(ctx, storage.KeyInMemoryStorage, m)
+	req := httptest.NewRequestWithContext(ctx, http.MethodGet, "/string/{flagKey}", nil)
+	req.SetPathValue("flagKey", "hello")
+	w := httptest.NewRecorder()
```

Comment:
See if you can DRY up the common setup behavior (while making sure the unit tests remain isolated) of these tests into subtests using [*T.Run](https://cs.opensource.google/go/go/+/refs/tags/go1.22.0:src/testing/testing.go;drc=76d90a34dd39b3e218e4c201d4497dca09c1cd6f;l=1703).